mycroft/skills/core.py

```diff
@@ -406,7 +409,7 @@ def show_text(self, text, title=None):
         self.clear()
         self["text"] = text
         self["title"] = title
-        self.show_page("SYSTEM_TEXTFRAME")
+        self.show_page("SYSTEM_TextFrame.qml")
 
     def show_image(self, url, caption=None, title=None):
```

Comment:
I think you forgot to add the fill as a parameter here.

```python
def show_image(self, url, caption=None, title=None, fill=None):
```

and then below
```python
        self["fill"] = fill or 'PreserveAspectFit'
```

to give a decent default action.

Reply:
fixed the missing parameter, the fill is by default set to stretch in qml so if there is none it will fallback to default